sat/cli/bootprep/input/base.py

```diff
@@ -242,6 +242,11 @@ def name(self):
         # items that inherit from BaseInputItem.
         return self.data['name']
 
+    @property
+    def boot_set(self):
```

Comment:
Why was  a `boot_set` property added on the `BaseInputItem`? This parent class applies to CFS configurations and BOS session templates in the input file, so it's not appropriate to add here. There is also already a `boot_sets` property on the `InputSessionTemplate` class where you needed to use it. Can we just use that one?